lsdcGui.py

```diff
@@ -3677,9 +3677,25 @@ def omegaTweakPosCB(self):
 
     def focusTweakCB(self,tv):
       tvf = float(tv)*daq_utils.unitScaling        
+
+      current_viewangle = daq_utils.mag1ViewAngle
+      if (self.zoom2Radio.isChecked()):
```

Comment:
Meta question - why are these "Radio"?

Reply:
you have only one view so selecting between the 4 zoom levels (Mag 1-4) makes sense. is that what you are asking?

also, the concept of buttons of which only one can be selected at a time has been called "radio buttons" for a long time - at least since the early HTML days